### DISCUSSION TITLE
Can I change the name of the domain beyong my GitHub username

### DISCUSSION THREAD

**Discussion by kristosh** [alshedivat/al-folio#1420] [Q&A] [answered]
I have set up a GitHub page for my own using the al-folio template. I have followed the instructions and now my page is deployed online. I have added it to the `config.yml` file the following details:

```
URL: 'https://username.github.io' # the base hostname & protocol for your site
baseurl: '/username/' # the subpath of your site, e.g. /blog/
```

In this way, my page is online and everything works smoothly, however, the `URL `is `https://username.github.io/username` instead of `https://username.github.io`. Moreover, the name of the repository is the name with my `GitHub username`.

In the instructions, it implied that the `baseurl `should be empty. When I remove it I have issues with the paths in the online version. Is there any way to remove the extra /username in my case? Also, can I just another name beyond the `username`?

**Comment by george-gca** [comment]
If you leave `baseurl` empty, it will use only `https://username.github.io/`. If you set `baseurl` then it will use `https://username.github.io/baseurl`. What errors exactly are you having?

**Comment by kristosh** [comment]
When I try your recommendation the site is portrayed properly locally after checking the localhost (by running `bundle exec jekyll serve --lsi`) but when I deploy the site online all the links/images are missing and the site URL is the same: `https://username.github.io/username`

**Comment by george-gca** [comment, marked as answer]
Just checked your page and you got it wrong. As can be seen [in the README file](https://github.com/alshedivat/al-folio#installation):

> If you plan to upload your site to <your-github-username>.github.io, note that the name of your repository must be `<your-github-username>.github.io` or `<your-github-orgname>.github.io`, as stated in the [GitHub pages docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites).

The name of your repo is just `[kristosh](https://github.com/kristosh)`. That is for a completely different thing as can be seen [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme).

**Reply by kristosh** [reply]
Wait the name of the repo just `kristosh` while the URL of the page then its` https://kristosh.github.io/kristosh`. If I modified the name of the repo to kristosh.github.io then the URL is getting very weird:  `https://kristosh.github.io/kristosh..github.io/`.

**Reply by george-gca** [reply]
I believe it is better if you start your site over again.

Look:
| site | repo name | `baseurl` in _config.yml |
| --- | --- | --- |
| https://george-gca.github.io/ | [george-gca.github.io](https://github.com/george-gca/george-gca.github.io) | |
| https://george-gca.github.io/multi-language-al-folio/ | [multi-language-al-folio](https://github.com/george-gca/multi-language-al-folio) | /multi-language-al-folio |
